src/main/java/mods/railcraft/common/util/inventory/StandaloneInventory.java

```diff
@@ -59,6 +62,12 @@ public StandaloneInventory(int size, @Nullable String name, @Nullable RailcraftT
         this.callback = callback == null ? null : new TileCallback(callback);
     }
 
+    public StandaloneInventory(int size, @Nullable String name, @Nullable ITrackKitInstance callback) {
```

Comment:
Why don't you just use the tile callback? You can get the tile from the track instance.

Reply:
I don't want to initialize the callback until the tile of the track kit is set. That code will look ugly.

Reply:
Hmm